react/components/ProductImages/components/ProductImage.tsx

```diff
@@ -24,7 +24,7 @@ interface Props {
 
 type AspectRatio = string | number
 
-const CSS_HANDLES = ['productImage', 'productImageTag']
+const CSS_HANDLES = ['productImage', 'productImageTag', 'productImageMain', 'productImageZoom']
```

Comment:
Don't forget to install the project dependencies so your pre-commit hooks are working properly 🙏

Reply:
@kaisermann which command should I use to perform these installations?

Reply:
A basic `yarn install` on the root folder will get it working!

Reply:
@kaisermann how can I solve this Lint problem on PR Actions that not worked well?

Reply:
I'm not sure why the check is not using the updated file, because it should be passing now. Anyway, you'd do it by running `yarn lint --fix`